akka-projection-grpc/src/main/protobuf/akka/projection/grpc/event_producer.proto

```diff
@@ -65,6 +65,10 @@ message InitReq {
   Offset offset = 4;
   // consumer defined event filters
   repeated FilterCriteria filter = 5;
+  // Used for Replicated Event Sourcing, the replica id of the consumer
+  string replica_id = 6;
+  // Used for Replicated Event Sourcing, other replicas that the consumer is connected to.
+  repeated string other_replica_ids = 7;
```

Comment:
@johanandren I have tried this instead. These replica ids are included from the consumer in the InitReq. Then that is used on the producer side to decide if an event based on its origin should be emitted or handled as FilteredEvent.

Reply:
For producer push it would be similar. These fields would be included in the ConsumerEventStart response from the consumer side and then used on the producer side.

Reply:
Ok, seems reasonable.